hare/eligibility/oracle_test.go

```diff
@@ -232,8 +232,7 @@ func TestCalcEligibility_EligibleFromTortoiseActiveSet(t *testing.T) {
 
 	numMiners := 5
 	o.mBeacon.EXPECT().GetBeacon(layer.GetEpoch()).Return(beacon, nil).Times(numMiners)
-	// there is no cache for tortoise active set. so each signature will cause 2 calls to GetBeacon()
-	o.mBeacon.EXPECT().GetBeacon(start.GetEpoch()).Return(beacon, nil).Times(2 * numMiners)
+	o.mBeacon.EXPECT().GetBeacon(start.GetEpoch()).Return(beacon, nil).Times(1)
```

Comment:
would it be correct to expect `numMiners + 1` calls here?

Reply:
they are using different parameters for GetBeacon

Reply:
of course, then ignore my comment 🙂